### DISCUSSION TITLE
Problems while initializing AR Session on mobile

### DISCUSSION THREAD

**Discussion by cristinategar** [Rufus31415/Simple-WebXR-Unity#53] [Q&A]
Hi!
I'm trying to load an immersive scene in mobile that loads elements from a blob (specifically videos and models) but at the moment of starting the session it crashes and doesn't initialize it. This does not happen to me if I load the elements from streamming assets, initializing the session and playing the video correctly. I tried a lot of things but I can't find any error in the browser console and i don´t know what else to do. These are the warnings shown when i try to enter the AR session.

[apache.arsoftarremoteassistance.com-1655987727539.log](https://github.com/Rufus31415/Simple-WebXR-Unity/files/8968373/apache.arsoftarremoteassistance.com-1655987727539.log)

Thank you in advance.

**Comment by Rufus31415** [comment]
It's hard to say what's going on. Did you try to compile in debug to see if there is more detail ?
From the traces, it seems to be a memory usage issue. Have you tried on Chrome on PC which is much more tolerant regarding memory management?

**Reply by cristinategar** [reply]
I tried to compile in debug and i don´t see anything else, just the same logs. Also, this only happens on Mobile. I've tried on PC, Hololens and Quest and it works.
